coderd/database/migrations/000128_drop_startup_logs_eof_and_add_completion.down.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+ALTER TABLE workspace_agents
+	DROP COLUMN started_at,
+	DROP COLUMN ready_at;
+
+ALTER TABLE workspace_agent_startup_logs ADD COLUMN eof boolean NOT NULL DEFAULT false;
```

Comment:
nit: Maybe leave a comment stating that "migrate down" won't bring back log entries with EOF = true.